lte/gateway/c/session_manager/SessionManagement.md

```diff
@@ -0,0 +1,56 @@
+# Session Management in Magma
+
+## Overview of SessionD
+SessionD is the main service responsible for managing and enforcing session 
+configurations. 
+
+### Interfaces
+SessionD interfaces with three distinct external components. 
+1. Access (MME, AAA) <br>
+ This component receives all connection and session related information from 
+ the UE. It also handles mobility management. But from SessionD's point of view,
+ it notifies SessionD whenever a UE is attached or detached. This notification 
+ will include any context that SessionD needs for enforcement.
+2. Policy (PolicyDB, FeG+PCRF/OCS) <br>
+ This component is responsible for propagating any session or policy 
+ configuration that SessionD must enforce. Whenever a new UE is introduced, 
+ SessionD will reach out to this component to fetch any configuration. Depending
+ on the configuration, SessionD maybe reach out to this component for updates. 
+3. Switch (PipelineD) <br>
+ This component is responsible for policy and QoS enforcement. It receives 
+ any relevant policy and QoS configuration from SessionD and periodically 
+ reports state accordingly.
+ 
+### Internal Architecture 
+![](SessionD_Lg_Arch.png)
+
+## Configurations (Not Exhaustive)
+- FeG Relay <br>
```

Comment:
maybe you should introduce `federated` and `non federated` concept in this bulletpoint, so you can use safely later